pkgs/tools/misc/xv/default.nix

```diff
@@ -1,21 +1,33 @@
-{ stdenv, fetchFromGitHub, rustPlatform, ncurses }:
+{ stdenv, lib, fetchFromGitHub, rustPlatform
+, ncurses ? null
+, darwin ? null }:
+
+let useNcurses = !stdenv.targetPlatform.isWindows; in
```

Comment:
After re-reading the manual sections on cross-compiling and specifying dependencies, I think this is supposed to be `hostPlatform`.